Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM alpine:3.5
+FROM alpine:3.10.3
 
 LABEL maintainer "Zappi DevOps <devops@zappistore.com>"
 
```

Comment:
Seems like other curator images have a few extra dependencies to support some features added to curator, see:
* https://github.com/bobrik/docker-curator/blob/master/Dockerfile
* https://github.com/elastic/curator/blob/master/Dockerfile

Reply:
> Seems like other curator images have a few extra dependencies to support some features added to curator, see:

We could, continue for now and add them on a need basis. The only dependencies that I've seen in the documentation is for the [`requests_aws4auth` python module if you plan on using Curator with AWS ES using IAM credentials](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/pip.html).

> * https://github.com/bobrik/docker-curator/blob/master/Dockerfile
> * https://github.com/elastic/curator/blob/master/Dockerfile

Oh, didn't know there's an official image for Curator. Maybe we can use that instead? Unfortunately I can figure out where it's hosted since it's not on https://www.docker.elastic.co. Thoughts? 🤔

Reply:
>We could, continue for now and add them on a need basis.

Fine with that, just wanted to inform everyone.

>Oh, didn't know there's an official image for Curator. Maybe we can use that instead? >Unfortunately I can figure out where it's hosted since it's not on https://www.docker.elastic.co. Thoughts?

There aren't any offical ones sadly, one of those is from the official repository but the maintainer said he has no interest in managing an official docker image.

Reply:
> Fine with that, just wanted to inform everyone.

Cool. Thanks.

> There aren't any offical ones sadly, one of those is from the official repository but the maintainer said he has no interest in managing an official docker image.

Ah, I see. Then we'll stick with ours.